public/admin/config.yml

```diff
@@ -1,3 +1,4 @@
+local_backend: true
```

Comment:
Usikker på om dette også vil fungere i produksjon. Brukes for å kunne kjøre [CMS lokalt](https://mong.github.io/#/netlifycms).